VideoLocker/res/layout/row_myvideo_course_list.xml

```diff
@@ -4,41 +4,39 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
 
-    <LinearLayout
+    <RelativeLayout
```

Comment:
Again, since we're nesting `RelativeLayout` containers here, one of them can be removed.